config/routes.rb

```diff
@@ -885,5 +885,21 @@ def healthcare_routes(window:)
     get :details
     get :actioncable
   end
+
+  if ENV['ENABLE_HMIS_API'] == 'true'
+    namespace :hmis_api, path: 'hmis-api', defaults: { format: :json } do
+      devise_for :users, class_name: 'HmisApiUser',
+                         skip: [:registrations, :invitations, :passwords, :confirmations, :unlocks, :password_expired],
+                         path: '', path_names: { sign_in: 'login', sign_out: 'logout' }
+
+      resources :user, only: [:none] do
+        get :index, on: :collection
+      end
+      resources :projects, only: [:none] do
+        post :index, on: :collection
+      end
```

Comment:
`GET /hmis-api/user` returns information about the currently logged in user. If no user is logged in, it returns an error response with a CSRF token. This allows the initial login POST request to be CSRF-protected (as discussed @eanders )

`POST /hmis-api/projects` returns a list of projects. It's just a dummy endpoint to test authentication and CSRF for POST requests.

Both of these endpoints should be replaced with the GraphQL endpoint.